yarn.lock

```diff
@@ -1385,9 +1385,9 @@
   resolved "https://registry.yarnpkg.com/@csstools/selector-specificity/-/selector-specificity-3.0.0.tgz#798622546b63847e82389e473fd67f2707d82247"
   integrity sha512-hBI9tfBtuPIi885ZsZ32IMEU/5nlZH/KOVYJCOh7gyMxaVLGmLedYqFN6Ui1LXkI8JlC8IsuC0rF0btcRZKd5g==
 
-"@demos-europe/demosplan-ui@github:demos-europe/demosplan-ui#f_set_demosplan-ui_invue3_compat_mode":
+"@demos-europe/demosplan-ui@github:demos-europe/demosplan-ui#i_vue3_migration":
```

Comment:
I assume, that should not be part of the PR

Reply:
Shouldn't it? The branch `f_set_demosplan-ui_invue3_compat_mode` doesn't exist anymore

Reply:
never mind